fail2ban/tests/actiontestcase.py

```diff
@@ -389,6 +389,51 @@ def testExecuteActionStartEmpty(self):
 		self.assertLogged('Nothing to do')
 		self.pruneLog()
 
+	def testExecuteWithVars(self):
+		self.assertTrue(self.__action.executeCmd(
+			r'''printf %b "foreign input:\n'''
+			r''' -- $f2bV_A --\n'''
+			r''' -- $f2bV_B --\n'''
+			r''' -- $(echo -n $f2bV_C) --''' # echo just replaces \n to test it as single line
+			r'''"''', 
+			varsDict={
+			'f2bV_A': 'I\'m a hacker; && $(echo $f2bV_B)', 
```

Comment:
shouldn't we just sanitize all the input we are getting from logs right away and replace any possibly dangerous symbol with some placeholder? e.g. `$` -> `+DOLLAR+`, ``````` -> `+BACKTICK+` etc so they could be used in regexps etc and also safely passed into the commands?